server/modules/mainServer.js

```diff
@@ -0,0 +1,114 @@
+/* Copyright (c) Trainline Limited, 2016. All rights reserved. See LICENSE.txt in the project root for license information. */
+'use strict';
```

Comment:
Now that the admin server is gone, would it make sense to just merge `mainServer.js` into `index.js`?